### DISCUSSION TITLE
Subdivs and Arnold

### DISCUSSION THREAD

**Discussion by sharktacos** [Autodesk/maya-usd#3374] [Ideas]
Currently when rendering in Arnold the subdiv settings on a USD mesh here:
<img width="376" alt="image" src="https://github.com/Autodesk/maya-usd/assets/23034637/50e88e1e-692b-47ec-84af-d67c71d17450">

 ...have no effect. Instead, the subdiv settings live under the mesh's Extra Attributes and are specific to Arnold:
<img width="589" alt="image" src="https://github.com/Autodesk/maya-usd/assets/23034637/63054e64-0246-4950-b605-e2761968cc10">

I'm hoping that it's on the roadmap to transition to supporting setting subdivs in the mesh settings. 

I've made this a discussion, as I was not entirely sure whether this would be considered as an issue for the maya-usd group or someplace else.

Maya 2024.1

**Comment by BigRoy** [comment]
I suspect this might be due to how Arnold USD interprets the USD data - and they've likely put these under their own arnold namespaced primvars. Might be a topic for [Arnold-USD](https://github.com/Autodesk/arnold-usd) instead.
